ui/packages/catalog-ui-search/src/main/webapp/component/navigation-middle/navigation-middle.view.js

```diff
@@ -14,17 +14,31 @@
  **/
 /*global require*/
 var Marionette = require('marionette')
-var template = require('./navigation-middle.hbs')
 var CustomElements = require('../../js/CustomElements.js')
-const router = require('../router/router.js')
+const properties = require('properties')
+import * as React from 'react'
+import { IntlProvider, FormattedMessage } from 'react-intl'
 
 module.exports = Marionette.LayoutView.extend({
-  template: template,
+  template(props) {
+    return (
+      <div className={props.menuClass}>
+        <IntlProvider locale="en">
+          <FormattedMessage
+            id="sourcesKeyword"
+            defaultMessage={`${props.menuText}`}
```

Comment:
✏️ Since there is no extra formatting, would `defaultMessage={props.menuText}` work?